Cargo.toml

```
@@ -39,6 +39,7 @@ syn = { version = "1.0.56", default-features = false, features = [
 ] }
 
 [dev-dependencies]
+pretty_assertions = "1.3.0"
```

```suggestion
pretty_assertions = "1"
```
Unless there is a specific reason to use at least 1.3.0, we shouldn't enforce it.
Are you gonna leave that in? Personally I don't mind.

---

yeah, it makes the assertions on the expansion output a lot more bearable.